CONTRIBUTING.md

```diff
@@ -26,9 +26,15 @@ Note: you will see warnings about `__dirname` and `require()` not being "a valid
 
 ```bash
 yarn build
-yarn --force
+yarn --force # only needed after very first build; afterward can be skipped
 ```
 
+#### Why is `yarn --force` needed?
+
+The advantages of using Lerna in a monorepo setup means when we run our tests (and even the [create-snowpack-app templates](./create-snowpack-app)), we’re using our local build of Snowpack and not npm’s published version. After all, if we had to publish to npm in order to test anything, that would be a bad experience for everyone!
+
+When we run `yarn build`, a binstub is built at `./snowpack/pkg/node-dist/index.bin.js`. This is what runs when you run `snowpack` in your CLI, as well as the thing that runs for all our tests. But say you ran `yarn build && yarn test` (no `yarn --force`), you’d get an error message: `/bin/sh: snowpack: command not found`. That’s because in all our tests and sub-projects it’s not enough for that to exist; that has to be symlinked inside every sub-project to `node_modules/.bin/snowpack` (again, because we want to use a local build and not npm’s version). `yarn` is the quickest way to symlink everything, however, `yarn --force` is required at this stage when dependencies are already installed. Without that flag Yarn (incorrectly) thinks there’s nothing to do. In other words, `yarn --force` is only needed **one time** as a one-line command to symlink everything for local development, after which only `yarn build` is needed for subsequent changes and rebuilds.
```

Comment:
"binstub" might be unclear, I never heard the term myself, maybe we can just leave it out and say the file is created?

Reply:
Maybe I’m using the wrong term, but I think it’s necessary to explain that `yarn --force` and symlinking is directly tied to the Snowpack executable (the [`bin` key in package.json](https://docs.npmjs.com/files/package.json#bin)). I think it’s important to understand that this isn’t just _any_ file, but one that’s necessary to execute Snowpack and is the whole reason behind this flow.

Maybe that link would help for “binstub”? Or what about replacing that term with “executable” instead? Both?

Reply:
I would like contributors to at least be familiar with the concept of Node executables, even if that means adding a new section in CONTRIBUTING rather than trying to define this concept through context.

Reply:
Maybe instead of 

> a binstub is built at

we could say 

> the `snowpack` binary file is built at

Reply:
>  the snowpack binary file is built at

is clearer to me, I got confused by binstub

Reply:
+1 to all this, and heard on that being a confusing term. This is completely pedantic, but I’m anticipating an “um, actually” because we never truly generate any binary files for Snowpack, so it’s inaccurate to call it that.

“Binstub” was meant to communicate “this isn’t binary; simply _binary-like_ in that a Node script has a [shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29) at the top.” But I think “executable script” is probably more familiar to folks while also being accurate. WDYT?